schema/article.schema.json

```diff
@@ -126,6 +126,15 @@
                     "translatable": true
                   }
                 },
+                "disabledText": {
```

Comment:
I think this should come _after_ `ariaLabel` as that is related to the button above

Reply:
Agreed. See [b21fbda](https://github.com/adaptlearning/adapt-contrib-trickle/pull/214/commits/b21fbdaba66024a17d2f6f1845f8c64f8c72e62d).